web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/ytelsefordeling/OmsorgOgRettDto.java

```diff
@@ -0,0 +1,29 @@
+package no.nav.foreldrepenger.web.app.tjenester.behandling.ytelsefordeling;
+
+import java.util.Set;
+
+import no.nav.foreldrepenger.behandlingslager.behandling.personopplysning.SivilstandType;
+import no.nav.foreldrepenger.behandlingslager.geografisk.Landkoder;
+import no.nav.foreldrepenger.web.app.tjenester.behandling.personopplysning.PersonadresseDto;
+
+public record OmsorgOgRettDto(Søknad søknad, RegisterData registerdata, ManuellBehandlingResultat manuellBehandlingResultat) {
+
+    public record Søknad(Boolean søkerHarAleneomsorg, String annenpartNavn, String annenpartIdent, Landkoder annenpartBostedsland,
```

Comment:
Skal du ha med rolle (Mor eller Far/Medmor) - eller tas det fra fagsakkontekst?

Reply:
hmm, trenger vi å vise det i panelet? Ser man ikke det ut i fra saken ellers?

Reply:
For aksjonspunktet så trengs rolle for å utlede om man skal spørre om uføre ... gjelder kun 14-14 bare far rett.
Antar at frontend henter den fra kontekst

Reply:
Hentes fra fagsakdto vil jeg tro